### PR TITLE
Cult communication is now innate and has an action button

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscdel: Cultists no longer communicate via tomes.
rscadd: Cultists now communicate via the 'Communion' action button.
/:cl:

icon

### CODE REVIEW

File: code/datums/mind.dm [hunk 1]
```diff
@@ -146,11 +146,7 @@
...
 	if(src in ticker.mode.cult)
-		ticker.mode.cult -= src
-		ticker.mode.update_cult_icons_removed(src)
-		var/datum/game_mode/cult/cult = ticker.mode
-		if(istype(cult))
-			cult.memorize_cult_objectives(src)
+		ticker.mode.remove_cultist(src, 0, 0)
 	special_role = null
 	remove_objectives()
 	remove_antag_equip()
```
Comment (ChangelingRain) [inline code/datums/mind.dm]: This entire fucking file is causing me physical pain

**Comment by coiax** [comment]
So cultists just get telepathy now?

**Comment by ChangelingRain** [comment]
Almost; all of the stuff that'd prevent you from using a tome also prevents comms use, so it's mostly a style thing; there are very few times cultists would lack a tome with robustin's tome-on-convert stuff.

**Comment by coiax** [comment]
It does take away stylistically though? Before you had to go to a dark corner, get your unholy tome out, and then commune with the others. Now you can chat to the cult while you're in the Hop line or eating a sandwich.

**Comment by ChangelingRain** [comment]
It still does the whispering, so it's not totally invisible.

**Comment by coiax** [comment]
Oh whispering is okay then. What's the range of whispering where you can hear that they're whispering, but not what they said?

**Comment by Zilenan91** [comment]
:+1:  Fuck tome comms this is better

**Comment by ChangelingRain** [comment]
It's like at a tile away where it's partial hearing

**Comment by Robustin** [comment]
I've only been able to test a single round of Oldman Cult but the fact that I saw at least two people use emergency comms despite there being a POPUP that basically says "THIS IS FOR EMERGENCY COMMUNICATION IF YOU CANNOT ACCESS A TOME, YOU ARE ABOUT TO FUCK YOURSELF UP, DO YOU WANT TO CONTINUE Y/N?" and both of those people had tomes available, convinces me that some people are hopelessly stupid, but it would be acceptable to sacrifice some cult flavor (I'm generally not a fan of giving non-lings a hivemind) just prevent the cult from getting fucked by an RD who's sitting there gushing blood out of his ass at roundstart.

**Comment by ChangelingRain** [comment]
It is frankly appalling how many people use emergency comms with a tome.

Also using the tome isn't really a lot of flavor, most of the flavor is in the messages themselves.

**Comment by Robustin** [comment]
When I say flavor I mean that you have to go to dark secluded places to commune with your cult, whereas now you can have cultists just sit there chit-chatting in broad daylight as long as they aren't standing directly next to someone.

**Comment by ChangelingRain** [comment]
Should I add a visible message, or would that be too much?

**Comment by Sarcalogo** [comment]
Just how casual are you going to make this already incredibly easy game mode? One good cultist before all these changes, could have carried the cult all by himself. As a cultist you can already:

Use an unavoidable full screen AOE stun
Use a full screen unavoidable AOE crit
Instantly recover anyone at any time who has been cuffed or killed
Teleport anywhere you want provided you or another has visited there once anytime you want
Teleport others anywhere you want provided you've been there once
Ghost and use meta information legally
Instantly revive any killed cultists with just a monkey cube box
Revive gibbed and incinerated players
Create impenetrable bases or prisons (provided nobody has a null rod)
See dead people while alive
Communicate with every single other person who is on your team in bright red letters if you ever need help.

You also get some pretty robust weapons and stuns all for free. If anything, Cult should be nerfed, it's incredibly easy.

**Comment by ChangelingRain** [comment]
This is why; www.ss13.eu/tgdb/tg/latest_stats.html#mode_cult

**Comment by Robustin** [comment]
While you're doing this, can you bump down the pop requirement to 25? I don't think its controversial given their win rate. With all the lavaland players ghosting its becoming really hard to even hit 30 players ready at roundstart.

**Comment by Sarcalogo** [comment]
You can revive gibbed players. You manifest them in front of a cloner, scan them, get off the rune and clone them from the records.

**Comment by ChangelingRain** [comment]
Ah, that. Honestly, that's probably an exploit but at least it's easy to fix if I bother.

**Comment by coiax** [comment]
Is that an exploit though? Or is it just ingenuity?

Fix is probably just to make the manifested ghost in some way that makes them unclonable. If pod person cloning also obeys that, then that'd be the fix.

**Comment by Robustin** [comment]
Its not an exploit, been that was since cult was created. If you manage to get exclusive control of a cloning pod without anyone intervening, go for it. Plus as anyone knows, cloning is slow as balls, it's a pretty weak strat to clone the manifested.

**Comment by optimumtact** [comment]
Don't respond to sarca posts

**Comment by Supermichael777** [comment]
make it an action button for the tome.

**Comment by ChangelingRain** [comment]
If I made it a button for the tome, it'd be usable without the tome in-hand unless I specifically prevented that, and then where are we?

**Pull request merged**